examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/ChimneyAdapter.java

```diff
@@ -0,0 +1,44 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class ChimneyAdapter extends ArrayAdapter<Chimney> {
+
+    private final String TAG = "ChimneyAdapter";
+
+    private List<Chimney> chimneys;
+    private Context context;
+
+    public ChimneyAdapter(Context context, List<Chimney> chimneys) {
+        super(context, -1, chimneys);
```

Comment:
-1 keeps the super method from trying to find a TextView.to populate. Since, it includes a picture Android wouldn't be able to auto populate the View.